5-Heroku/server/middleweres/cookies.js

```diff
@@ -0,0 +1,21 @@
+
+const {v1:uuidv1} = require('uuid');
+const ToDoModel = require('../models')
+
+
+const checkCookies = function(req, res, next) {
+    if (!req.cookies) {
+        const userId = uuidv1();
+        res.cookie("userId", userId);
+        createNewUser(userId);
```

Comment:
why are you doing it here? i'd expect this middleware to be more generic and not to involve concerns of the app itself (saving an empty todo etc)